app/views/companies/show.html.haml

```diff
@@ -76,16 +76,15 @@
                                          items_count: @items_count
 
 
-    .row.center
-      %hr
-      %p
-        - if user_can_edit
-          = link_to "#{t('companies.edit_company')}",
-                    edit_company_path(@company), class: 'btn btn-secondary edit-company'
-
-        - if current_user.admin?
-          = link_to "#{t('.delete')}", @company, method: :delete,
-                    class:'btn btn-danger delete-company',
-                    data: { confirm: "#{t('.confirm_are_you_sure')}" }
+    %hr
+    .text-right
```

Comment:
This was a consious decision on my part. Because: those two pages where this is used (application and company show) are fundamentally different. There is edit and delete. The others dont have that.

Reply:
** Not Resolved** 
Is it a stopping requested change or can we address this separately?